schemas/definitions.json

```diff
@@ -346,6 +346,9 @@
                 "$ref": "#/definitions/valuesNumberValue"
               }
             }
+          },
+          "meta": {
```

Comment:
I don't understand how this would work - a property named `meta` with nothing but a `required` field - this does not reference the meta definition at all?

Reply:
Oh never mind, I think I got it: commonValueFields pulls in the meta def.